docs/Configuration.md

```diff
@@ -0,0 +1,88 @@
+# OSS Review Toolkit Configuration
+
+This page describes the different configuration options of ORT.
+
+## Repository Configuration
+
+ORT can be configured for a specific repository by adding an `.ort.yml` file to the root of the repository. Currently
```

Comment:
That sounds a bit as if you can only scan the repository if a `.ort.yml` file is present. Maybe better: "Repository-specific custom configuration options for ORT can be set via a `.ort.yml` file that is added to the root of the repository".